src/commands/start.js

```diff
@@ -66,21 +65,18 @@ async function start(opts: *) {
   // Run `adb reverse` on Android
   if (opts.platform === 'android') {
     const args = `reverse tcp:${opts.port} tcp:${opts.port}`;
-    const adb = process.env.ANDROID_HOME
-      ? `${process.env.ANDROID_HOME}/platform-tools/adb`
-      : 'adb';
 
     try {
-      await exec(`${adb} ${args}`);
+      await exec(`adb ${args}`);
```

Comment:
we can also simplify it to:

```
const command = `adb reverse tcp:${opts.port} tcp:${opts.port}`
```

and use the variable `command` everywhere instead of `adb ${args}`

Reply:
Good point!